### PR TITLE
Fix invalid axes aliases

### PR DESCRIPTION
**Pull request by t-bltg**
Fix https://github.com/JuliaPlots/Plots.jl/issues/3783.

### CODE REVIEW

**Comment by BeastyBlacksmith** [comment]
Can you add a test for this?

**Comment by t-bltg** [comment]
> Can you add a test for this?

I've already added a test: https://github.com/JuliaPlots/Plots.jl/pull/3803/commits/08d2157790ffde8753bdfda95230eaa23a68c5d3

File: test/test_axes.jl [hunk 1]
```diff
@@ -63,3 +63,17 @@ end
     @test twpl[:top_margin] == 2Plots.cm
     @test twpl[:bottom_margin] == 2Plots.cm
 end
+
+@testset "aliases" begin
+    plot(1:2, xl = "x label")
```
Comment (BeastyBlacksmith) [inline test/test_axes.jl:68]:
```suggestion
    @test plot(1:2, xl = "x label")[1][:xaxis][:guide] == "x label"
```
And the others alike

Comment (t-bltg) [reply test/test_axes.jl:68]: Woops, incomplete. Thanks.

**Comment by t-bltg** [comment]
Tests fixed.

File: test/test_axes.jl [hunk 2]
```diff
@@ -63,3 +63,28 @@ end
...
+    p = plot(1:2, xl = "x label")
+    @test p[1][:xaxis][:guide] === "x label"
+    p = plot(1:2, xrange = (0, 3))
+    @test xlims(p) === (0, 3)
+    p = plot(1:2, xtick = [1.25, 1.5, 1.75])
+    @test p[1][:xaxis][:ticks] == [1.25, 1.5, 1.75]
+    p = plot(1:2, xlabelfontsize = 4)
+    @test p[1][:xaxis][:guidefontsize] == 4
+    p = plot(1:2, gridalpha = .07)
+    @test p[1][:xaxis][:gridalpha] ≈ .07
```
Comment (BeastyBlacksmith) [inline test/test_axes.jl:77]: Does `gridalpha` only set the value for the x-axis or all grids?

Comment (t-bltg) [reply test/test_axes.jl]: Sorry, it was an un-voluntary change, I used `xgα = ...` in my initial test. Fixed.

**Review by BeastyBlacksmith — APPROVED** [review]
Should be good to go when tests pass

**Pull request merged**